roles/setup_replication/tasks/upstream_node.yml

```diff
@@ -7,7 +7,7 @@
   when:
     - hostvars[inventory_hostname].upstream_node_private_ip is defined
     - node.private_ip == hostvars[inventory_hostname].upstream_node_private_ip
-  loop: "{{ lookup('edb_devops.edb_postgres.pg_sr_cluster_nodes', wantlist=True) }}"
```

Comment:
I would suggest to keep using the lookup function here. If we want to include only this tasks file, `pg_cluster_nodes` wont be defined.

Reply:
fixed